### PR TITLE
Re:#9966 Fix Crash in MZFlickable

### PR DESCRIPTION
**Pull request by strseb**
Soooo i did set up a 6.8 dev build and was able to repro the crash seen in #9966 - this is a fun one. 
In 6.8.0 we get the "onHeightChange" before the window has finished to initialize, so that when we call ` mapToItem(window.contentItem)` we defer a nullptr and crash.

### CODE REVIEW

File: extension/socks5proxy/bin/CMakeLists.txt
Comment (strseb) [inline extension/socks5proxy/bin/CMakeLists.txt:1]: Qt 6.8.0 no longer links to this it seems so we need to manually declare that :)

File: src/commands/commandui.cpp
Comment (strseb) [inline src/commands/commandui.cpp:1]: This errored and i think noone is using it, so let's just remove it.

File: src/platforms/windows/daemon/windowsfirewall.cpp
Comment (strseb) [inline src/platforms/windows/daemon/windowsfirewall.cpp:1]: qAsConst is depricated in 6.8.0

File: extension/socks5proxy/bin/CMakeLists.txt
Comment (oskirby) [reply extension/socks5proxy/bin/CMakeLists.txt:1]: I think we tend to handle this using pragmas: `#pragma comment(lib, "Iphlpapi")`

**Review by oskirby — APPROVED** [review]
Looks good.

Just one minor nitpick about how to link libraries on Windows, but it's not worth holding the patch up if you feel strongly about it.

**Comment by XavierCLL** [comment]
Hi @strseb 

I am not sure if the purpose of this PR is fix all the problems with Qt6.8, I applied this patch (also this one 9961) to the stable version, and the UI is usable, but one thing is not working, when I click in the list of servers "Select location", the UI crash with the following traceback:

[trace.txt](https://github.com/user-attachments/files/17532325/trace.txt)

**Comment by strseb** [comment]
Hey @XavierCLL - i was not expecting one small patch to magically fix all issues that might arise with 6.8 :) 
Given that we're still shipping the client on most plattforms with qt6.6 i really appreciate i really appreciate the testing, posting traces and bug's filed :) 
Let's pull that stacktrace into a new bug and we'll tackle it from there.

**Pull request merged**